prow/cmd/hook/main_test.go

```diff
@@ -70,3 +71,11 @@ func TestOptions_Validate(t *testing.T) {
 		}
 	}
 }
+
+func TestParseConfig(t *testing.T) {
```

Comment:
Why?

Reply:
I cannot tell you anymore.I had a reason, probably not a good or even an invalid one. I can remove that.